src/styles/zoomwindow.scss

```diff
@@ -0,0 +1,53 @@
+@import "variables.scss";
+
+.zoomwindow {
+  background: white;
+  position: fixed;
+  left: 5px;
+  top: 5px;
+  z-index: 1;
+  width: 25px;
+  height: 25px;
+  padding-right:40px;
+  padding-left:5px;
+  align-content: center;
+  border-width: 3px;
+  border-style: solid;
+  border-color: $navbar-color;
+  margin: 3px;
+  margin-top: 50px;
```

Comment:
a lot of these have shorthand notation. for example, look at https://developer.mozilla.org/en-US/docs/Web/CSS/margin

Reply:
^